terraform/environments/hmpps-domain-services/locals_test.tf

```diff
@@ -188,6 +188,180 @@ locals {
         }
       }
     }
+
+    baseline_lbs = {
+      networklb = {
```

Comment:
probably want to call it something other than networklb.  We've gone with "private" in other apps because we couldn't think of anything better